pkg/model/manifest.go

```diff
@@ -805,10 +883,34 @@ func (m *Manifest) setDefaults() error {
 	return nil
 }
 
-func (m *Manifest) mergeWithOktetoManifest(other *Manifest) {
-	if len(m.Dev) == 0 && len(other.Dev) != 0 {
-		m.Dev = other.Dev
+func (m *Manifest) mergeWithOktetoManifest(other *Manifest) []string {
+	warnings := []string{}
+	buildWarnings := m.Build.merge(other.Build)
+	warnings = append(warnings, buildWarnings...)
+
+	dependenciesWarnings := m.Dependencies.merge(other.Dependencies)
+	warnings = append(warnings, dependenciesWarnings...)
+
+	deployWarnings := m.Deploy.merge(other.Deploy)
+	warnings = append(warnings, deployWarnings...)
+
+	m.Destroy.merge(other.Destroy)
+
+	devWarnings := m.Dev.merge(other.Dev)
+	warnings = append(warnings, devWarnings...)
+
+	if other.Context != "" && other.Context != m.Context {
```

Comment:
I don't think so, we set the namespace and context when reading the file. Since we are not validating/setting the defaults on the second one `other.Context` can be empty and we should only show a warning if one of the dependencies has one of these fields set with another value

Reply:
I was saying this because it may be an opportunity to show the user that there is information in his "secondary" manifest that is incorrect or at least will not be processed

Reply:
it's correct, if they launch it directly from the second one, but we need to manage both cases